Source/RapyutaSimulationPlugins/Private/Core/RRUObjectUtils.cpp

```diff
@@ -12,6 +12,101 @@
 #include "Core/RRMathUtils.h"
 #include "Core/RRMeshActor.h"
 
+static TArray<uint8> DATA_SYNTH_CUSTOM_DEPTH_STENCILS = []()
+{
+    TArray<uint8> data = {
```

Comment:
what's this for?

Reply:
They are hardcoded Segmentation mask values due to incorrect pixel capturing in UE5 (like the ones read from output png are different from the values written into the stencil buffer) as mentioned in the description

Reply:
They are hardcoded Segmentation mask values due to incorrect pixel capturing in UE5 (like the ones read from output png are different from the values written into the custom depth buffer) as mentioned in the description